datalad/support/tests/test_sshconnector.py

```diff
@@ -74,3 +74,26 @@ def test_ssh_manager_close():
 
     ok_(not exists(opj(manager.socket_dir, 'localhost')))
     ok_(not exists(opj(manager.socket_dir, 'datalad-test')))
+
+
+@with_tempfile(mkdir=True)
+@with_tempfile
```

Comment:
with_tempfile doesn't create the file, so tests fail.  You can use e.g. `@with_tempfile(content="blah")` to have it created.